packages/spartan-council/test/integration/Cast.test.js

```diff
@@ -178,9 +178,9 @@ describe('ElectionModule (cast)', function () {
             ElectionModule.getVotePower(voter3.address),
           ]);
 
-          assertBn.equal(debts[0], ethers.BigNumber.from('127506055884648544164502'));
-          assertBn.equal(debts[1], ethers.BigNumber.from('144460521479569515200508'));
-          assertBn.equal(debts[2], ethers.BigNumber.from('122875067482594009777896'));
+          assertBn.gt(debts[0], 0);
```

Comment:
Checking if the votePower is greater than 0 is enough here?

Reply:
Updated the code again to go back to real values check.